airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlCdcProperties.java

```diff
@@ -69,7 +69,7 @@ private static Properties commonProperties(final JdbcDatabase database) {
     if (sourceConfig.get("replication_method").has("server_time_zone")) {
       final String serverTimeZone = sourceConfig.get("replication_method").get("server_time_zone").asText();
       if (!serverTimeZone.isEmpty()) {
-        props.setProperty("database.serverTimezone", serverTimeZone);
+        props.setProperty("database.connectionTimeZone", serverTimeZone);
```

Comment:
Can you add a link to the exact section in Debezium docs where this property is listed

Reply:
https://debezium.io/documentation/reference/stable/connectors/mysql.html
`The time zone will be queried from the server by default. If this fails, it must be specified explicitly by the database connectionTimeZone MySQL configuration option. For example, if the database’s time zone (either globally or configured for the connector by means of the connectionTimeZone option) is "America/Los_Angeles", the TIMESTAMP value "2018-06-20 06:37:03" is represented by a ZonedTimestamp with the value "2018-06-20T13:37:03Z".`

Reply:
> If this fails
Do we know for a fact there was a failure?

Is the failure here because the server timezone is not IANA compliant?
That is `CEST` rather than `Europe/Berlin`